mag/src/main/java/com/ca/mas/core/http/MAGRequest.java

```diff
@@ -238,6 +245,17 @@ public MAGRequestBuilder responseBody(MAGResponseBody responseBody) {
         }
 
         /**
+         * The request is being made outside of primary gateway.
+         * When the value is set to true, all automatically injected credentials in SDK will be excluded in the request.
```

Comment:
Do we want to change the comment here since we don't set the value to true by parameter?  "When this method is called, all automatically injected credentials in the SDK will be excluded in the request."